libs/client/src/client/PezzoOpenAI.ts

```diff
@@ -8,6 +8,7 @@ import {
   ReportData,
 } from "../types";
 import { Pezzo } from "./Pezzo";
+import { merge } from "lodash";
```

Comment:
as discussed, better write a merge util and have it in the client package, this way we don't force users to install lodash